src/GenesisWindow.cpp

```diff
@@ -111,6 +111,8 @@ GenesisWindow::GenesisWindow() :
 	menu->AddItem(new BMenuItem(LANGS("SUBMENU_DELETE"),	new BMessage(MENU_COMMANDS_DELETE), 0));
 	menu->AddSeparatorItem();
 	menu->AddItem(new BMenuItem(LANGS("SUBMENU_CREATELINK") , new BMessage(MENU_COMMANDS_CREATE_SYMLINK), 'L'));
+	menu->AddSeparatorItem();
+	menu->AddItem(new BMenuItem(LANGS("SUBMENU_TERMINAL"),	new BMessage(MENU_TERMINAL), 0));
```

Comment:
You wanna add a OPT+ALT+T shortcut for that? That'd be the same as for the "Open Terminal" Tracker add-on. It's in my muscle memory :)

Also probably useful: ALT+H for home folder and ALT+D for Desktop in the "cd" popup menu.

Finally, there are two spaces in front of the second "new" in this line.